cg/meta/workflow/nf_analysis.py

```diff
@@ -266,7 +266,9 @@ def get_deliverables_template_content() -> list[dict]:
         """Return deliverables file template content."""
         raise NotImplementedError
 
-    def get_deliverables_for_case(self, case_id: str) -> PipelineDeliverables:
+    def get_deliverables_for_case(
```

Comment:
Does this function. have tests? It would be good to include a test that checks that the correct files are skipped

Reply:
included